renderer/suncg/scene.hh

```diff
@@ -46,7 +47,8 @@ class SUNCGScene : public ObjSceneBase {
     explicit SUNCGScene(
         std::string obj_file,
         std::string model_category_file,
-        std::string semantic_label_file);
+        std::string semantic_label_file,
+        float minDepth = 0.3);
```

Comment:
Define constant at top (to make more salient)?

I guess this also isn't exposed in the higher level APIs, but that might not matter

Reply:
It's better if we can have some configuration file that includes these parameters. But I don't have bandwidth to do that. 
If you do need to frequently change minDepth from the caller, I'm fine with adding an argument. Otherwise a constant is probably much simpler.

Reply:
I added it as an argument to indicate to future contributors that this is a parameter of some consequence (setting it to DEFAULT NEAR of 0.1 would sacrifice a lot of the 16bits to mostly empty space), and we might want to expose to higher-level APIs in the future.